tests/unit/obuf-limits.tcl

```diff
@@ -70,4 +70,32 @@ start_server {tags {"obuf-limits"}} {
         assert {$omem >= 100000 && $time_elapsed < 6}
         $rd1 close
     }
+
+    test {Won't write replies if client output buffer hard limit is enforced} {
+        r config set client-output-buffer-limit {normal 100000 0 0}
+
+        # Total size of all items must be more than 2 * 100000
+        for {set i 0} {$i < 100000} {incr i} {
+            r lpush mylist $i $i
+        }
```

Comment:
i don't like to run such a large number of commands without a pipeline, the round trip can slow down the tests. we can use deferring client and set up the limit after it.
or better yet, why not just push 200 elements of 1kb string each?

Reply:
OK, i will push big size elements